.all-contributorsrc

```diff
@@ -265,10 +265,16 @@
       "avatar_url": "https://avatars.githubusercontent.com/u/202806705?v=4",
       "profile": "https://github.com/lea-m-truss",
       "contributions": ["bug", "code", "maintenance", "review"]
+    },
+    {
+      "login": "aligg",
+      "name": "Ali Glenesk",
+      "avatar_url": "https://avatars.githubusercontent.com/u/7331495?v=4",
+      "profile": "https://github.com/aligg",
+      "contributions": ["code", "maintenance"]
```

Comment:
Let's let the bot take care of this:

https://github.com/trussworks/react-uswds/pull/3153